### DISCUSSION TITLE
position: top and position: bottom?

### DISCUSSION THREAD

**Discussion by lyoungblood14** [Turbo87/sidebar-v2#323] [Q&A] [answered]
I am working on an app and had this question on stack overflow: [link](https://stackoverflow.com/questions/77867564/making-the-active-part-or-view-of-the-map-smaller-than-the-actual-height-of?noredirect=1#comment137275741_77867564) 

Someone suggested having a look at this sidebar-v2 plugin. I think it seems that it could accomplish what I'm looking for (that the map centers next to its sibling divs, but then still appears to be under them, thus also allowing the leaflet controls to be moved by the sibling divs. However, as a relative beginner, I'm overwhelmed. Is it worth trying to understand if it is possible to make this sidebar into a position: top and bottom? I would need two at once.   

Has anyone tried this/can maybe offer a bit of guidance? Or which part of the source code could be helpful here? @Turbo87 🙏

**Comment by Falke-Design** [comment, marked as answer]
What do you mean with ***make this sidebar into a position: top and bottom***?

You can only place it on the left or right side.

**Reply by lyoungblood14** [reply]
Exactly. I need pretty much the exact same functionality, but as topbar and bottombar. I will dive into the code to see if I can figure out what exactly makes it shift the leaflet-bar over when the sidebar is opened (e.g. the zoom controls in the first example https://turbo87.github.io/sidebar-v2/examples/index.html). Thanks for the quick reply.
